sdk/ml/azure-ai-ml/tests/test_configs/flows/flow_with_meta/compare.md

```diff
@@ -0,0 +1,29 @@
+| name                | ?              | ? - conflict with UX?                         | Y                                              | ? - it's shared by all runs? | N                     |
+| version             | Y              | ?                                             | ? - do we have concept version for flow in UX? | N                            | N                     |
+| display_name        | ?              | ?                                             | Y                                              | N - not a run config?        | N                     |
+| tags                | ?              | ?                                             | ? - do we have concept tags for flow in UX?    | N - not a run config?        | N                     |
+| runtime             | ?              | N - not part of flow definition               | N - not meta?                                  | Y                            | N                     |
+| node_variant        | ?              | N - not part of flow definition               | N - not meta?                                  | Y                            | N                     |
+| columns_mapping     | Y              | N - not part of flow definition               | N - not meta?                                  | Y                            | N                     |
+| connections         | Y              | N - not part of flow definition               | N - not meta?                                  | Y                            | N                     |
+| is_deterministic    | Y              | N - we don't have such concept in promptflow? | N                                              | N                            | N                     |
+
+conclusion:
+
+| filed name          | place                                                        |
+| ------------------- | ------------------------------------------------------------ |
+| additional_includes | flow.dag.yaml                                                |
+| name                | flow.dag.yaml/ignore(can be passed via public interface of `load_component`) |
+| version             | ignore                                                       |
+| display_name        | the same to `name`                                           |
+| tags                | the same to `name`                                           |
+| runtime             | run.yaml                                                     |
+| node_variant        | run.yaml                                                     |
+| columns_mapping     | run.yaml                                                     |
+| connections         | run.yaml                                                     |
+| is_deterministic    | ignore (can be passed via private interface of `load_component`) |
```

Comment:
we might aslo need to add environment as office user may want to use their own image, etc..

Reply:
e.g. user compoennt yaml:
https://eemo.visualstudio.com/TEE/_git/TEEGit?path=/Offline/MSAIData/python/components/QRSLLM/module_spec.yaml&version=GBdevelop

Reply:
may add a column describe all the comamnd/parralle component fields.